tests/pds/registrysweepers/test_ancestry.py

```diff
@@ -0,0 +1,136 @@
+import itertools
+import os.path
+import unittest
+from typing import Dict
+from typing import List
+
+from pds.registrysweepers import ancestry
+from pds.registrysweepers.ancestry import AncestryRecord
+
+from tests.mocks.registryquerymock import RegistryQueryMock
+
+
+class AncestryFunctionalTestCase(unittest.TestCase):
+    input_file_path = os.path.abspath("./tests/pds/registrysweepers/test_ancestry_mock_registry.json")
```

Comment:
Is it possible to use `pkg_resources` here? The `./` makes running tests dependent on `$CWD`.

(And maybe that's okay! I'm trying to think of a time—whether on the developer desktop or in the Roundup Action—when $CWD wouldn't be `==` `git clone` dir.)

Reply:
You're almost certainly right - will fix

Reply:
slightly more complex than expected - shelving for now